Libraries/Text/TextInput/Singleline/RCTUITextField.m

```diff
@@ -85,10 +85,10 @@ @implementation RCTUITextField {
   return [UIFont systemFontOfSize:17];
 }
 
-static UIColor *defaultPlaceholderTextColor()
+static RCTUIColor *defaultPlaceholderTextColor()
 {
   // Default placeholder color from UITextField.
-  return [UIColor colorWithRed:0 green:0 blue:0.0980392 alpha:0.22];
+  return [RCTUIColor colorWithRed:0 green:0 blue:0.0980392 alpha:0.22];
```

Comment:
Missing the TODO comment

Reply:
This change is already inside a ` // [TODO(macOS ISS#2323203)` ... ` // ]TODO(macOS ISS#2323203)` block.